src/scss/components/_v-tooltip.scss

```diff
@@ -26,9 +26,11 @@ $ec-tooltip-width-value: 5px;
     }
 
     .ec-tooltip--bg-bright & {
+      @include box-shadow-level-1;
```

Comment:
I wouldn't add this line, we have another modifier for it: `.ec-tooltip--shadow`.

Reply:
Not sure if is correct to have a modifier as we can have only 1 type of tooltip and 1 type of popover on ebo. The only type of tooltip we can have comes with a shadow. Shall I remove the modifier?

Reply:
I wouldn't remove that, these components should follow specs in frontify and not what EBO needs. I disagree with removing functionality which we already have built because it's not in MVP for EBO.

BTW, I see 3 types of tooltips on EBO, the dark one in navigation (no shadows), the dark one in the submitted requests (with a shadow) and the bright one in the footer of the table (with the shadow).

Reply:
Hi team! As we discussed in Slack, the version on black tooltip with a shadow is an error from InVision, I've already updated that screen. We only have the two scenarios that @stavros-tomas defined on the thread :)